avaframe/out3Plot/plotUtils.py

```diff
@@ -54,7 +56,7 @@
 matplotlib.rcParams['figure.titlesize'] = cfg['titleSize']
 matplotlib.rcParams['figure.dpi'] = float(cfg['figResolution'])
 matplotlib.rcParams['figure.autolayout'] = True
-
+ls = ['-', '--', '-.']
```

Comment:
Isn't this the default order of linestyles anyways?

Reply:
If I want to force the style of multiple lines to be the same, I think I need this